src/controllers/client/list.controller.ts

```diff
@@ -0,0 +1,24 @@
+import { Request, Response } from 'express';
```

Comment:
@GarfDev You had implemented a POST /claims/info controller to basically handle a Read operation. I have changed this to align better with the REST spec. 

Look into the REST specification to understand how to name endpoints appropriately:
https://www.w3.org/2001/sw/wiki/REST